src/CommonDBTM.php

```diff
@@ -6710,4 +6710,27 @@ public static function displayList(
             'as_map'             => false,
         ]);
     }
+
+    /** @return static[] */
+    public static function getSeveralFromDBByCrit(
```

Comment:
I'm not really fan of the name; but also I have nothing better to propose right now :D